lib/screens/NewPatientScreen.dart

```diff
@@ -359,10 +381,23 @@ class _NewPatientFormState extends State<_NewPatientForm> {
     }
     return _makeQuestion('Phone Number',
         child: TextFormField(
+          decoration: InputDecoration(
+            prefixText: '+266',
+          ),
           controller: _phoneNumberCtr,
+          onEditingComplete: () {
+            _phoneNumberCtr.text = _formatPhoneNumber(_phoneNumberCtr.text);
```

Comment:
Also call `_formatPhoneNumber(_phoneNumberCtr.text)` in the `_onSubmitForm` method, otherwise the unformatted phone number will be stored.